app/controllers/book_controller.rb

```diff
@@ -0,0 +1,10 @@
+class BookController < ApplicationController
+  include Wor::Paginate
```

Comment:
you can include this in the ApplicationController, you're probably going to need pagination in more controllers, so if you inherit that it's better